templates/pioneer-citp-news-disinfo/measurements/measurements.1.schema.json

```diff
@@ -25,7 +26,18 @@
                     "referrerDomain": { "type": "string" },
                     "dayOfWeek": { "type": "integer" },
                     "timeOfDay": { "type": "integer" },
-                    "pageCategory": { "type": "integer" },
+                    "pageCategory": {
+                        "type": "integer",
+                        "description": "deprecated, use classifierResults"
+                    },
+                    "classifierResults": {
+                        "type": "object",
+                        "properties": {},
```

Comment:
Sorry, the generated schema looks wrong, I believe the properties field needs to go away for this to be converted into a key-value pair.

```suggestion
```